packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs/BubbleGraph/Reflow/MultipleDatasets.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import BubbleMultipleDataset from '@cerner/terra-graphs/lib/components/Bubble/BubbleMultipleDataset';
+import Button from 'terra-button/lib/Button';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+import '@cerner/terra-graphs-docs/lib/Css/ExampleGraphContainer.module.scss';
+import getSimpleAxisData from '@cerner/terra-graphs-docs/src/example-datasets/graphConfigObjects/Bubble/simplePanningAxisData';
+import bubbleDataBasic from '@cerner/terra-graphs-docs/src/example-datasets/dataObjects/Bubble/bubbleDataBasic.js';
+import bubbleDataCustomRadius from '@cerner/terra-graphs-docs/src/example-datasets/dataObjects/Bubble/bubbleDataCustomRadius.js';
+
+/*
+Please refer to the documentation below to see the graphConfig and data objects
+*/
+const dataset = utils.deepClone(bubbleDataBasic);
+const dataset1 = utils.deepClone(bubbleDataCustomRadius);
```

Comment:
nit: rename to `dataset1` and `dataset2`
or
`basicDataset` and `customRadiusDataset`

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/163/commits/55d5aeb126b4ac5dce4f6c7eea41f1e759ee042f#diff-618a78fc59bba4a43ea9dffdc15179c3bb8664b6452367e5437296a9e228b537R13).